### ISSUE TITLE
Latest 1.7.6.3 APACHE image does not build

### ISSUE THREAD

**Issue by matks** [PrestaShop/docker#201]
https://travis-ci.org/PrestaShop/docker/builds/639875885

```
configure: error: unrecognized options: --with-freetype-dir, --with-jpeg-dir
The command '/bin/sh -c apt-get update 	&& apt-get install -y libmcrypt-dev 		libjpeg62-turbo-dev 		libpcre3-dev 		libpng-dev 		libfreetype6-dev 		libxml2-dev 		libicu-dev 		libzip-dev 		default-mysql-client 		wget 		unzip     && rm -rf /var/lib/apt/lists/*     && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/     && docker-php-ext-install iconv intl pdo_mysql mbstring soap gd zip' returned a non-zero code: 1

The command "docker build -t "prestashop/base:$BASE" base/images/$BASE" exited with 1.

0.06s$ docker tag prestashop/base:$BASE prestashop/base:5.6-apache
Error response from daemon: No such image: prestashop/base:apache

The command "docker tag prestashop/base:$BASE prestashop/base:5.6-apache" exited with 1.

59.51s$ docker build -t "$image" images/$VERSION
Sending build context to Docker daemon  2.048kB
Step 1/5 : FROM prestashop/base:5.6-apache
5.6-apache: Pulling from prestashop/base
```

**Comment by itsvahid** [comment]
i had the same problem. 
looks like this solves the it:
https://github.com/docker-library/php/issues/912#issuecomment-559918036

**Comment by matks** [comment]
> i had the same problem.
> looks like this solves the it:
> [docker-library/php#912 (comment)](https://github.com/docker-library/php/issues/912#issuecomment-559918036)

Thanks ! It seems someone else than me fixed it as 1.7.6.4 apache image is built successfully https://travis-ci.org/github/PrestaShop/docker/builds/658718068?utm_medium=notification&utm_source=github_status

**Comment by matks** [comment]
Since issue seems fixed I close the issue. Thanks again for the help @itsvahid
